operator/pkg/helmreconciler/prune.go

```diff
@@ -68,6 +68,7 @@ var (
 	// AllClusterResources lists all cluster scope resources types which should be deleted in purge case, including CRD.
```

Comment:
This is "cluster scoped" but the the yaml we set `namespace`. Seems one is incorrect, not sure which

Reply:
It is indeed a namespaced one, not sure then where to put it. I think it should only be removed when purging, as it's mostly static and might be reused across different revisions...

Reply:
hmm now that I looked more, I don't think we ever make any usage of it that is only "Cluster scoped". So may be enough to just rename it..?
Should double check my assumption though

Reply:
do you mean rename the `AllClusterResources` variable? To something like `AllResources`?